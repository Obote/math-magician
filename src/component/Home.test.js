import React from 'react';
import renderer from 'react-test-renderer';
import Home from './Home';

test('should render', () => {
  const tree = renderer.create(<Home />).toJSON();
  expect(tree).toMatchSnapshot();
});
