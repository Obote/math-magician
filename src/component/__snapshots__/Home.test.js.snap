// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render 1`] = `
<div>
  <h2>
    Welcome to the Home Page
  </h2>
  <p>
    This is the home page of our website.
  </p>
</div>
`;
